lib/commonmark.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 var MarkdownIt = require('markdown-it');
-var sanitizer = require('sanitizer');
-var uriPolicy = require('./uriPolicy').uriPolicy;
+var DOMPurify = require('dompurify');
+var { linkPolicy, addTabindexToIdPolicy } = require('./policies.js');
```

Comment:
Let's hide the `dompurify` into a local `./sanitize.js` file completely with no separate policies file.

```suggestion
var sanitize = require('./sanitize.js');
```

Reply:
moved to sanitize.js